charts/postgres-operator/templates/configmap.yaml

```diff
@@ -25,4 +25,6 @@ data:
 {{ toYaml .Values.configLoggingRestApi | indent 2 }}
 {{ toYaml .Values.configTeamsApi | indent 2 }}
 {{ toYaml .Values.configConnectionPooler | indent 2 }}
+  sidecars:
+    {{ toYaml .Values.configSidecars | indent 4 }}
```

Comment:
have you tested specifying sidecars in a configmap? I doubt it supports nested objects. I think one can use it only with the config CRD. But haven't tested myself yet.

Reply:
No, I have not. I am just using the operatorConfig.